### DISCUSSION TITLE
placental transfer

### DISCUSSION THREAD

**Discussion by HaiXxxxxx** [Open-Systems-Pharmacology/Forum#1812] [Questions & Problems]
Hi,
I know three different methods for informing the transplacental passive diffusion clearance (Dpl) and partition coefficient between the fetal and maternal compartment (Kf,m) of placental transfer, but I don't get the method to estimate the Dpl and Kf,m from the ex vivo cotyledon perfusion experiment. Can you give me some help calculating these two parameters? 
Thanks for the help in advance.
Kind regards,
Zhihai

**Comment by AndreDlm** [comment]
Dear @caozhihai,

to estimate these parameters you need to conduct a parameter identification using a compartmental model and observed data from the ex vivo cotyledon perfusion experiment. 

Several models have been published in the literature (e.g., [Mian et al.](https://www.frontiersin.org/articles/10.3389/fped.2021.733520)) and a MoBi model has been shared [here](https://github.com/Open-Systems-Pharmacology/Pregnancy-Models/tree/master/CotyledonPerfusionModel). When using this model, you need to adjust the compound-specific parameter values (e.g., molecular weight, lipophilicity, pKa, fraction unbound etc.) and potentially also few system-specific parameters (e.g., volumes of the maternal and fetal reservoirs, flow rates used in the cotyledon perfusion experiment). Next, load in your observed data from the cotyledon perfusion experiment, set up a parameter identification and fit Dpl and Kf,m. 
Be careful with non-identifiability issues/parameter correlations when fitting these two parameters together - you can evaluate that by checking the option `Calculate Sensitivity` in the `Configuration` tab of the parameter identification.

When integrating the fitted values in a PBPK model make sure to upscale Dpl from one cotyledon to the whole placenta, e.g., via the volume. Further information can be found in the article by [Mian et al.](https://www.frontiersin.org/articles/10.3389/fped.2021.733520).

Best regards
André

**Reply by HaiXxxxxx** [reply]
Dear@AndreDlm,
Thank you for your reply. I adjusted the compound-specific parameter values and system-specific parameters. The result obtained after my parameter identification is shown in the figure below. What is the cause of this?
Thanks for the help in advance.
Best regards,
Zhihai
![QQ截图20240809083802](https://github.com/user-attachments/assets/62e9bd23-1c02-4bd2-82c7-377303fad0b4)

**Reply by AndreDlm** [reply]
Dear @caozhihai,
It's difficult to identify the reason for the misfit without knowing the model in detail. The initial concentrations in the maternal reservoir are substantially overestimated which could indicate that the dose or volume is not correctly specified. You might want to check these parameters, especially the volumes for maternal and fetal reservoir which can vary considerably as discussed [here](https://github.com/Open-Systems-Pharmacology/Forum/discussions/1704#discussioncomment-8527329)

**Reply by HaiXxxxxx** [reply]
Dear @AndreDlm 
Thank you. The volumes for maternal and fetal reservoir are 200 ml according to the ex vivo cotyledon perfusion experiment. I enlarged the maternal reservoir and got the same results.
Thanks for the help in advance.
Best regards,
Zhihai

**Reply by AndreDlm** [reply]
Dear Zhihai,
Something seems to be still wrong with the dose or volume. A few things to check would be:
Is the dose correctly implemented (no conversion error from mg to mol)? The reservoir volume should be similar to the dosage divided by the initial concentration of the drug, is that the case?
If you believe that the model is correctly set up and the observed data are correct, you could check alternative hypotheses: Did the compound fully dissolve in the medium of the perfusion assay or was there a potential solubility limitation (precipitation)? Is there any reason to assume strong metabolism of the compound in the trophoblast? Does the compound bind to cellular components or is there reason to assume substantial accumulation in the trophoblast? Does the compound strongly bind to any experimental/analytical material (plastic tubes etc)?

**Reply by HaiXxxxxx** [reply]
Dear @AndreDlm ,
I modified the solubility from 7*E-6 cm/min to the default value. After running the simulation, I got the following result below. However, after parameter identification, I got a similar result from the previous picture. Thank you.

**Reply by AndreDlm** [reply]
Dear Zhihai, I could not find the results you are referring to in your previous post. The solubility value above has speed as unit (cm/min) which would be more consistent for a diffusion coefficient. I suggest going through the different points mentioned above to pin down the potential error.

**Reply by HaiXxxxxx** [reply]
Dear @AndreDlm ,
Sorry, I typed the wrong word. It's permeability, not solubility. What does this value describe and how is it obtained?
Thank you.

**Reply by AndreDlm** [reply]
Typically, diffusional drug transfer between two compartments is described by a modified Fick's law of diffusion where the permeability coefficient (*P*) is multiplied with the surface area of the membrane (*SA*) to obtain the permeability ("transfer clearance", unit: volume/time), similar like that:
![image](https://github.com/user-attachments/assets/fb871597-4c13-48fa-ae5e-d784797d466f)

where N is molar drug amount, fu the drug's unbound fraction, K the partition coefficient, and the subscripts cell and int refer to the two compartments between which diffusion occurs. 

In the equation above, the permeability coefficient is used for describing diffusion in both directions (A-->B and B-->A). However, there are also equations in which different permeability coefficients are used for the two directions.

It can be obtained, e.g., through parameter identification or based on in vitro experiments.

In your specific case it is difficult to say how the permeability (coefficient) is exactly used. This depends on the differential equation (specified in the transport building block) and path of the parameter (probably specified in the spatial structure building block).

**Reply by HaiXxxxxx** [reply]
Dear @AndreDlm,
Thank you very much for your help. I solved.

**Reply by HaiXxxxxx** [reply]
Dear @AndreDlm,
Sorry to bother you again. Can you show me how to convert the optimized values to Kf,m and Dcot?
Thank you.

**Reply by AndreDlm** [reply]
Dear Zhihai, assuming you want to integrate the fitted values in a whole-body PBPK model as done by [Mian et al.](https://www.frontiersin.org/articles/10.3389/fped.2021.733520) you can directly use the fitted value for Kf,m in the PBPK model. The fitted value for Dcot is only valid for a single cotyledon so it needs to be up-scaled to the whole placenta. This can be done either via the volume (see e.g., Eq. 6 in [De Sousa Mendes](https://pubmed.ncbi.nlm.nih.gov/26518984/)):
D<sub>pl</sub> = D<sub>cot</sub> * V<sub>placenta</sub>/V<sub>cot</sub> 
Alternatively, via the surface area of the fetal villi (as done in [Mian et al.](https://www.frontiersin.org/articles/10.3389/fped.2021.733520))

**Reply by wanxiang227** [reply]
Dear@AndreDlm @caozhihai,
Sorry to bother you. Where do we enter these three fitted values(Partition coefficient, permeability factor cell-F_pls-M and permeability factor cell-M_pls-F) in the PBPK model. The value of Kf,m in the CotyledonPerfusionModel in git-hub is 4.32, why is it 0.737 in the article. 
Thanks.

**Reply by AndreDlm** [reply]
In the article various values for Kf,m were tested:
![image](https://github.com/user-attachments/assets/5c5fe667-0ce0-4178-a768-f43e350a4080)

The permeability factor cell-F_pls-M (f<sub>out</sub>) and permeability factor cell-M_pls-F (f<sub>in</sub>) were used as a coefficient to modulate the transfer clearance, see Eq. 11 in the article:
![image](https://github.com/user-attachments/assets/86bf2bbe-586e-4272-9bf6-0fb26d8b0850)

You could also remove these factors (or fix them to 1) and, e.g., overwrite the product of permeability * SA with your upscaled value.

**Reply by wanxiang227** [reply]
Dear @AndreDlm 
Thank you. After obtaining Dpl and Kf,m by three methods, it is simulated in the model established in PK-SIM by sending Mobi, which corresponds to the following parameters?
![image](https://github.com/user-attachments/assets/ccc8a1e1-b41c-4166-98c1-e84fb96816cc)
